superset/sql_lab.py

```diff
@@ -52,25 +50,13 @@
 from superset.utils.dates import now_as_float
 from superset.utils.decorators import stats_timing
 
-
-# pylint: disable=unused-argument, redefined-outer-name
-def dummy_sql_query_mutator(
```

Comment:
Note I don't believe moving this is to `config.py` is a breaking change since any custom logic would already be defined in a custom config override.